stack.yaml

```
@@ -1,3 +1,3 @@
-resolver: lts-18.18
+resolver: lts-18.24
```

```suggestion
resolver: lts-18.18
```

It's easier to leave this at the same number as the one in wire-server.  Otherwise lots of redundant re-building of world.